pom.xml

```diff
@@ -27,6 +27,7 @@
     <java.level>8</java.level>
     <jenkins.version>2.150.1</jenkins.version>
     <jcasc.version>1.36</jcasc.version>
+    <useBeta>true</useBeta>
```

Comment:
I'm sorry, but this is a blocker for me. Mailer is vastly used and I don't see how we could depend on API that could be removed without warnings.

Reply:
AFAICT this can be removed (since the field is read using reflection and defaulting to ADMINISTER if it fails).